gnovm/cmd/gno/test.go

```diff
@@ -246,6 +254,10 @@ func execTest(cfg *testCfg, args []string, io commands.IO) error {
 			io.ErrPrintfln("FAIL    %s \t%s", pkg.Dir, dstr)
 			io.ErrPrintfln("FAIL")
 			testErrCount++
+			if cfg.failfast {
+				io.ErrPrintfln("FAIL")
+				return fmt.Errorf("FAIL: %d build errors, %d test errors", buildErrCount, testErrCount)
+			}
```

Comment:
`-failfast` does not fail just on the first failing package, but on the first failing test, so this implementation is incomplete

Reply:
Fair enough, good catch :)
fixed here I think 2be439e1ac73208153a53e5c43ab488708c49ab9